### DISCUSSION TITLE
FC indicators need to have high visual priority for player satisfaction

### DISCUSSION THREAD

**Discussion by rozukke** [ppy/osu#26571] [Ideas]
With the current discussion around letter grades for the standard gamemode, there are a few camps rising regarding whether to move forward with the current system in Lazer or to revert back to Stable grading. Personally, I think that the idea of an FC indicator (as mentioned by peppy) is the best way forward alongside a more generous grade distribution (S ranks starting around the 95% or 96% mark with no additional requirements). In comparison to Stable, where an A rank encompasses everything from a 1x miss 99% play to a 92% accuracy play, it would provide much needed variety.

I believe that a majority of people who are reluctant to change to the new system are people who have scores that are visually distinguished by the presence of the S rank as an identifier of an FC. As a result, shifting the S rank requirements down devalues their plays. In order to solve this conflict, there needs to be a visually distinct element introduced that would add instant value back to FC plays.
I've already noticed some ideas like this in the Figma repository of designs by Flyte:
![image](https://github.com/ppy/osu/assets/73006620/bf005ec5-f53d-4047-9190-6330ed2619c0)
Accessible via [this link](https://www.figma.com/file/EjS27IBaZhc2qX97lFBUxN/Song-Select-5?type=design&node-id=0-1&mode=design).
Here I presume that the gradient colors on the score represent additional info about it, like whether or not it is 100% accuracy or if it is an FC; I'm sharing it here just so that more people see what it might look like. I would even increase the visibility of the gradients to make them stand out more, or include them in a more overt way, though I am not a professional designer by any means.

I would, imo, encourage anyone with a say in terms of visual design to move in this direction, or at least to poll the community over whether something like this would add sufficient visual weight to satisfy them and replace the current S-rank = FC paradigm. If more people were aware of what this might look like, they would be a lot more keen to move forward with the new system, so it is to some degree a problem of communication.

I think many people would also be interested to hear about the current plans for the FC indicators and whether anyone else has ideas regarding how to solve this debate.

**Comment by bdach** [comment]
> Here I presume that the gradient colors on the score represent additional info about it, like whether or not it is 100% accuracy or if it is an FC

I don't think that's what that is. I don't think that gradient is supposed to mean *anything* and I think trying to make it so would be opaque and wrong. Whether the score is an FC is indicated by the max combo display being green, as it is on the website.

**Reply by rozukke** [reply]
I do agree that this concept can be done better, and I did mention that I would personally make it more visible, such as with my favorite solution, which is just to color FC grades with a rainbow color, so rainbow S'es, rainbow A's, etc. The green combo color is all well and good, but there's likely to be less player resistance if their impressive plays are visually distinct and very front and center.

**Comment by MasicoreLord** [comment]
What if the S rank has an FC requirement only on classic client or w/ classic mod applied?
Meaning lazer both can keep the new S rank threshold and classic modded/classic client plays will retain the old requirements for S.
_cross-posted from Discord s rank thread_

**Comment by rozukke** [comment]
For reference, I've attached a proof of concept design that I think would be a pretty decent way to represent more impressive scores visually (this would probably be for fcs). The "aggressiveness" of it adds to the visual distinction, and I think people would be happy with the amount of emphasis it provides.
![image](https://github.com/ppy/osu/assets/73006620/58140771-9ea1-4afb-9bc2-d2e9c72b9fde)

I am not a professional designer by any means so if it looks too extreme, I'm sure someone that knows what they're doing more than me can do a better job.

**Reply by felipemarins** [reply]
I think a golden glow highlight around the rank as frenzibyte suggested on discord may be a better option as it seems more compatible with the overall score card redesign. But design may be a very subjective thing, so it doesn't mean your design is bad as a concept. I just think that it feels too out of place if considering the surrounding elements colour scheme.
